tests/ui/test_search.py

```diff
@@ -53,8 +58,87 @@ def test_sorting_by(base_url, selenium, category, sort_attr):
 def test_incompative_extensions_show_as_incompatible(base_url, selenium):
     page = Home(selenium, base_url).open()
     term = 'Ui-Addon-Android'
-    items = page.search_for(term)
+    items = page.search.search_for(term)
     for item in items.result_list.extensions:
         if term == item.name:
             detail_page = item.click()
             assert detail_page.is_compatible is False
+
+
+@pytest.mark.nondestructive
+def test_search_suggestion_term_is_higher(base_url, selenium):
+    page = Home(selenium, base_url).open()
+    term = 'Ui-Addon'
+    items = page.search.search_for(term, execute=False)
+    for count, item in enumerate(items):
+        if item.name == term:
+            assert count < 1
+
```

Comment:
Don't we have a finite number of items we expect here? It'd be better to explicitly test them instead of a `for` loop that wouldn't fail in case we only see one item (for whatever reason).

Also, I don't quite get that loop, isn't that almost equivalent to just writing ``assert items[0].name == term``? `count < 1` will only be true for the very first entry.